addons/rook/1.4.9/install.sh

```diff
@@ -153,6 +153,17 @@ function rook_cluster_deploy_upgrade() {
         return 0
     fi
 
+    echo "Awaiting rook-ceph operator"
+
+    if ! spinner_until 600 rook_version_deployed ; then
```

Comment:
This code only needs to run on upgrades. It is currently blocking new rook installations.